aten/src/ATen/native/README.md

```diff
@@ -170,6 +170,27 @@ Note that argument type modifiers such as defaults and optional are not currentl
 
 The declarations also support the following attributes:
 
+**Overloads.** You can register multiple functions with the same name and different
+function signatures if you give them unique overload names. An overload name
+is specified after the function name, separated by a dot.
+
+Overload names do not have to be globally unique, but must be unique in the set
+of all overloads for the same function. Please try to make overload names
+semantically meaningful. An overload name that just enumerates the argument
```

Comment:
You should expound on what makes a name semantically meaningful.  Just saying something like "the current standard is to name the first new parameter name or type" would be helpful.